### PR TITLE
Bug keyboard events replays

### PR DESCRIPTION
**Pull request by buh**

### CODE REVIEW

File: Sources/UI/Extensions/Keyboard.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ import RxGesture
 struct Keyboard {
     
     static let shared = Self()
-    let notification: Driver<KeyboardNotification>
+    let notification: Observable<KeyboardNotification>
```
Comment (b-onc) [inline Sources/UI/Extensions/Keyboard.swift]: Can you explain why this change was needed?

File: Sources/UI/User Story/Chat/Chat View Controller/ChatViewController.swift [hunk 2]
```diff
@@ -141,7 +141,7 @@ open class ChatViewController: ViewController, UITableViewDataSource, UITableVie
         changesEnabled = true
         setupFooterUpdates()
         
-        Keyboard.shared.notification.drive(rx.keyboard).disposed(by: disposeBag)
+        Keyboard.shared.notification.bind(to: self.rx.keyboard).disposed(by: self.disposeBag)
```
Comment (b-onc) [inline Sources/UI/User Story/Chat/Chat View Controller/ChatViewController.swift]: Why do we need explicit `self` here?

Comment (buh) [reply Sources/UI/User Story/Chat/Chat View Controller/ChatViewController.swift]: No needed. I'll remove it.

**Comment by buh** [comment]
Fixed and added a couple minor commits.
